core/services/ocrcommon/data_source.go

```diff
@@ -294,31 +293,30 @@ func (ds *inMemoryDataSourceCache) updateCache(ctx context.Context) error {
 	ds.mu.Lock()
 	defer ds.mu.Unlock()
 
-	// check for any errors
-	_, latestTrrs, latestUpdateErr := ds.executeRun(ctx)
```

Comment:
This checked for any errors in the pipeline results, which doesn't make sense since multiple bridge data sources are used and not always all of them are successful